barretenberg/cpp/src/barretenberg/stdlib/goblin_verifier/goblin_recursion.test.cpp

```diff
@@ -1,85 +0,0 @@
-#include "barretenberg/goblin/goblin.hpp"
```

Comment:
This test no longer has value for us. It was essentially an in-test implementation of the UltraGoblinVanillaIVC which we recently deleted. It had odd syntax, was incomplete, and did not test anything meaningful that isn't tested elsewhere.